examples/resources/squadcast_team_role/import.sh

```diff
@@ -1,2 +1,2 @@
-# teamID:roleID
-terraform import squadcast_team_role.test 62d2fe23a57381088224d726:62da76c088f407f9ca756ca5
\ No newline at end of file
+# teamID:teamRole(ex: Admin, User, Observer)
+terraform import squadcast_team_role.example_resource_name 62d2fe23a57381088224d726:Admin
```

Comment:
@punit-squadcast It should be an ID right? Not Role name?

Reply:
no for role import we need to give name, I tried with Id but it was throwing error, it was working fine with name.